bin.js

```diff
@@ -4,13 +4,22 @@ var proc = require('child_process')
 var os = require('os')
 var path = require('path')
 
+var targetMismatch = (process.env.npm_config_platform && process.platform !== process.env.npm_config_platform) ||
```

Comment:
can we make this guard against process being a thing also, ie `typeof process !== 'undefined' && process.env` in case this runs in weird envs?

Reply:
(unsure if we rely on process else where tho)

Reply:
We briefly had something like that (https://github.com/prebuild/node-gyp-build/commit/8cc5a84a1b53b4ba04c9c5d42e35bee4b787bbfb) then you reverted: https://github.com/prebuild/node-gyp-build/commit/28b562f593ed2d0f3a1dcca4ee781f36d4b20e21.

That situation hasn't changed: if someone needs a guard against undefined `process` (or nested properties like `process.versions`) then we can fix that across the board, but for now it isn't necessary.